superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -71,7 +71,11 @@ export default function AddDataset() {
   >(datasetReducer, null);
 
   const HeaderComponent = () => (
-    <Header setDataset={setDataset} datasetName={dataset?.dataset_name ?? ''} />
+    <Header
+      setDataset={setDataset}
+      title={dataset?.table_name ?? 'New dataset'}
```

Comment:
do you want this title to be 'New dataset' if dataset.table_name is undefined or ''?

Reply:
Oh good point, I didn't think of undefined table names. How would that table show up in the left panel, in that case? Would it be a blank line? Maybe a good default would be "Undefined dataset"?

Reply:
Actually, depending on the intended header behavior, we might need to figure something else out. If I set the default here to "Undefined dataset" it will temporarily change to that once a schema is chosen and a table name isn't selected. This could be okay as well though, because _technically_ it is undefined at that point. What do you think?

![2022-09-23 18 35 06](https://user-images.githubusercontent.com/55605634/192069663-ead646e8-bb12-42d7-ac22-e7362f886cbc.gif)

Reply:
Ok, what you have is good!